### DISCUSSION TITLE
Using docker image apache/doris:build-env-for-1.1.0 compile error

### DISCUSSION THREAD

**Discussion by bingfenglai** [apache/doris#12204] [A - General / Q&A]
error: '/var/local/thirdparty/installed/lib64/libhs.a', needed by 'src/service/doris_be', missing and no known rule to make it.

How to solve this problem?

**Comment by dataalive** [comment]
update docker image or execute `unset  DORIS_THIRDPARTY` to recompile the thirdparty lib
